class.jetpack.php

```diff
@@ -4882,6 +4947,17 @@ public static function register() {
 			),
 			'timeout' => $timeout,
 		);
+
+		list( $activation_source_name, $activation_source_keyword ) = get_option( 'jetpack_activation_source', false );
+		
+		if ( $activation_source_name ) {
+			$args['_as'] = urlencode( $activation_source_name );
+		}
+
+		if ( $activation_source_keyword ) {
+			$args['_ak'] = urlencode( $activation_source_keyword );
+		}
+
```

Comment:
These three lines (ok, 7, if we count the wraps :p) could probably be extracted into a function since they're repeated 3 times :p

Reply:
I extracted two instances into a shared function because they were both inside the same class.

Reply:
🎉